src/doc/trpl/compound-data-types.md

```diff
@@ -47,8 +47,8 @@ This pattern is very powerful, and we'll see it repeated more later.
 
 There are also a few things you can do with a tuple as a whole, without
 destructuring. You can assign one tuple into another, if they have the same
-contained types and arity. Tuples have the same arity when they have the same
-length.
+contained types and [arity](glossary.html#arity). Tuples have the same arity
+when they have the same length.
```

Comment:
Small matter, but while we're at it: regarding the preference for reference-style linking, it seems not very widely known, but every Markdown parser I've used—including the reference impl I believe—allows this to be written as `[arity]` in the prose instead of `[arity][]` with empty brackets. Is that too magical or surprising? Any opinion?